x-pack/legacy/plugins/ml/server/models/data_recognizer/modules/apm_jsbase/ml/decreased_throughput_jsbase.json

```diff
@@ -0,0 +1,36 @@
+{
+  "job_type": "anomaly_detector",
+  "groups": [
+    "apm"
+  ],
+  "description": "APM JSBase: Identifies periods during which the application is processing fewer requests than normal.",
+  "analysis_config": {
+    "summary_count_field_name": "doc_count",
+    "bucket_span": "15m",
+    "detectors": [
+      {
+        "detector_description": "low_count",
+        "function": "low_count"
+      }
+    ],
+    "influencers": [
+      "service.name"
+    ]
+  },
+  "allow_lazy_open": true,
+  "analysis_limits": {
+    "model_memory_limit": "10mb"
+  },
+  "data_description": {
+    "time_field": "@timestamp"
+  },
+  "custom_settings": {
+    "custom_urls": [
+      {
+        "url_name": "APM",
+        "time_range": "2h",
+        "url_value": "apm#/services/$service.name$/transactions?rangeFrom=$earliest$&rangeTo=$latest$&refreshPaused=true&refreshInterval=0&kuery=&transactionType=request"
```

Comment:
Where there is no `service.name` influencer, we need to remove `$service.name$` from the URL if possible. Currently it is keeping `$service.name$` in the URL:

![image](https://user-images.githubusercontent.com/7405507/71681775-81cfc900-2d85-11ea-93a0-e52bf5d25f5b.png)

Reply:
Check https://github.com/elastic/kibana/pull/53792/commits/9f8347d905f19b0ef3e611d13b80900c317b88d4